core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -2009,6 +2022,26 @@ public Row newRow() {
     @Override
     public Row newRow(long timestamp) {
         switch (rowAction) {
+            case ROW_ACTION_NO_PARTITION:
```

Comment:
nit: I belive the old cases order was optimised to have the most frequent branches first.

Reply:
we might want to re-test, but i think diff will be negligible if any. Java optimises switch statement for keys in ascending order, which is what intellij suggested to adhere to.